src/charts/axis.js

```diff
@@ -215,6 +219,18 @@ export default function Axis (_container) {
     }
   }
 
+  function shouldXLabelsRotate () {
+    const APPROX_FONT_WIDTH = 5
```

Comment:
This constant could move up, since it's also used in getNumberOfLabelsToSkip

Reply:
Agreed, I think I meant to do that but forgot.